libraries/AP_Mount/AP_Mount_CADDX.h

```diff
@@ -0,0 +1,55 @@
+/*
+  CADDX mount using serial protocol backend class
+ */
+#pragma once
+
+#include "AP_Mount_config.h"
+
+#if HAL_MOUNT_CADDX_ENABLED
+
+#include "AP_Mount_Backend_Serial.h"
+#include <AP_HAL/AP_HAL.h>
+#include <AP_Math/AP_Math.h>
```

Comment:
```suggestion
```

Reply:
removed include, thanks

Reply:
Actually I've put this back because AP_Math.h is where "quaternion" is included

Reply:
```
diff --git a/libraries/AP_Mount/AP_Mount_CADDX.h b/libraries/AP_Mount/AP_Mount_CADDX.h
index fe423f55459..3d831209bf7 100644
--- a/libraries/AP_Mount/AP_Mount_CADDX.h
+++ b/libraries/AP_Mount/AP_Mount_CADDX.h
@@ -8,8 +8,7 @@
 #if HAL_MOUNT_CADDX_ENABLED
 
 #include "AP_Mount_Backend_Serial.h"
-#include <AP_Math/AP_Math.h>
-#include <AP_Common/AP_Common.h>
+#include <AP_Math/quaternion.h>
 
 class AP_Mount_CADDX : public AP_Mount_Backend_Serial
 {
```

Reply:
ok thanks.  Updated to remove AP_Common.h and change to <AP_Math/quaternion.h>.  I'm surprised we don't need AP_Common.h in there.  We include that in every other AP_Mount backend so I guess we could remove it from many places

Reply:
> ok thanks. Updated to remove AP_Common.h and change to <AP_Math/quaternion.h>. I'm surprised we don't need AP_Common.h in there. We include that in every other AP_Mount backend so I guess we could remove it from many places

Probably, yes - and that will save compile time, which everybody likes.

We're getting away with some shenanigans by having those all around the place - things like the stdint.h include come to mind :-)